### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by szimeonovn**

### CODE REVIEW

File: _i18n/en/_posts/2019-08-27-press-release.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 # layout: news
-title: Press release Aug 2019
+title: Press release - CYBERG strikes new international agreements
```
Comment (zsapkagy) [inline _i18n/en/_posts/2019-08-27-press-release.md:3]: Do we need "Press release" here ❓ 
This was the docs.label before and yesterday we have to remove it 🤔

File: _includes/local-date.html [hunk 2]
```diff
@@ -1,9 +1,23 @@
 <span class="date">
-  {% if site.lang == 'hu' %} {% assign m = include.date | date: "%-m" %} {{
-  include.date | date: "%Y" }}. {% case m %} {% when '1' %}január {% when '2'
-  %}február {% when '3' %}március {% when '4' %}április {% when '5' %}május {%
-  when '6' %}június {% when '7' %}július {% when '8' %}augusztus {% when '9'
-  %}szeptember {% when '10' %}octóber {% when '11' %}november {% when '12'
-  %}december {% endcase %} {{include.date | date: "%-d"}}. {% else %}
-  {{include.date | date: "%-d %B %Y"}} {% endif %}
+    {% if site.lang == 'hu' %}
+        {% assign m = include.date | date: "%-m" %}
```
Comment (zsapkagy) [inline _includes/local-date.html:3]: 🙏

File: _i18n/en/_posts/2019-08-27-press-release.md (hunk 1 above)
Comment (szimeonovn) [reply _i18n/en/_posts/2019-08-27-press-release.md:3]: Yep Erik asked for it in the pdf

Comment (zsapkagy) [reply _i18n/en/_posts/2019-08-27-press-release.md:3]: OK

**Pull request merged**